libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -121,12 +121,19 @@ public UserIdentity generateUserIdentity(
 
     public Optional<IdentityClaim> findIdentityClaim(List<VerifiableCredential> vcs)
             throws HttpResponseExceptionWithErrorBody, CredentialParseException {
-        return findIdentityClaim(vcs, true);
+        return findIdentityClaimPrivate(vcs, true);
     }
 
     public Optional<IdentityClaim> findIdentityClaim(
             List<VerifiableCredential> vcs, boolean checkEvidence)
             throws HttpResponseExceptionWithErrorBody, CredentialParseException {
+        return findIdentityClaimPrivate(vcs, checkEvidence);
+    }
+
+    // Need to have a private method for tests ?
```

Comment:
?

Reply:
`BuildProvenUserIdentityDetailsHandlerTest` > `when(mockUserIdentityService.findIdentitytClaim(any())).thenCallRealMethod()` only calls the first and last lines of each method it need to call... Why this happens I don't know but suspect that's why there was a private method there before my changes

I accept it's a bit hacky, so this could be a follow-up piece of work to tidy up

Reply:
That sounds real suspicious :/

If you do need to keep this here, then can we at least tidy up the comment to explain more clearly?

Reply:
I'll try to remove the .thenCallRealMethod() and instead just mock the response